models/balancer/ethereum/balancer_v1_ethereum_liquidity.sql

```diff
@@ -0,0 +1,112 @@
+{{
+    config(
+        schema='balancer_v1_ethereum',
+        alias='liquidity',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['day', 'pool_id', 'token_address'],
+        post_hook='{{ expose_spells_hide_trino(\'["ethereum"]\',
```

Comment:
this post_hook can be the `expose_spells` macro, rather than `expose_spells_hide_trino` as it's materialized as a table. if proceeding as-is, then the table won't appear on the dune explorer under dunesql db. you can see both macros in the macro directory.